action/protocol/execution/evm/evm.go

```diff
@@ -303,8 +303,11 @@ func processSGD(ctx context.Context, sm protocol.StateManager, execution *action
 	if execution.Contract() == action.EmptyAddress {
 		return nil, 0, nil
 	}
-
-	receiver, percentage, ok, err := sgd.CheckContract(ctx, execution.Contract())
+	height, err := sm.Height()
+	if err != nil {
+		return nil, 0, err
+	}
+	receiver, percentage, ok, err := sgd.CheckContract(ctx, execution.Contract(), height-1)
```

Comment:
if height == 0, `height - 1` will overflow

Reply:
a good reminder, but it is a hard-fork after Quebec at least, so height=0 will not occur.

Reply:
> if height == 0, `height - 1` will overflow

validate the height >= 1 before calling CheckContract